_overviews/scala3-book/ca-context-bounds.md

```diff
@@ -7,6 +7,7 @@ num: 61
 previous-page: ca-given-using-clauses
 next-page: ca-given-imports
 ---
+<span class="tag tag-inline">Scala 3 only</span>
```

Comment:
Context bounds feature in both Scala 2 and 3 so I think this tag should be removed and tabs be included for both scala 2 and 3

Reply:
perhaps the issue is with the phrase "context parameter" used throughout - maybe this needs more thought

Reply:
I think in the plan we agreed that the Scala Center would work on that page.

Reply:
> I think in the plan we agreed that the Scala Center would work on that page.

Oops, I’m sorry about that.

Reply:
No worries! It was probably not very clear :)